pymc/distributions/discrete.py

```diff
@@ -635,12 +635,10 @@ def logcdf(value, mu):
         safe_mu = at.switch(at.lt(mu, 0), 0, mu)
         safe_value = at.switch(at.lt(value, 0), 0, value)
 
-        res = (
-            at.switch(
-                at.lt(value, 0),
-                -np.inf,
-                at.log(at.gammaincc(safe_value + 1, safe_mu)),
-            ),
+        res = at.switch(
```

Comment:
Do we need a test, or does it not need to be updated?

Reply:
I mentioned above, I didn't add a test because we are not doing any test like this for any of the other logcdf methods